### PR TITLE
Redirect to /dashboard when successfully logged in

### PR DESCRIPTION
**Pull request by babanesma**
**Is your PR related to a specific issue/feature? Please describe and mention issues.**

Successful log in was causing an infinite redirection 

```
*[Warning] [Vue Router warn]: Detected an infinite redirection in a navigation guard when going from "/" to "/login". Aborting to avoid a Stack Overflow. This will break in production if not fixed. (vue-router.js, line 43)
```

**Describe alternatives you've considered**

Change `onGuestOnly` to `onLogin` to redirect logged in users to /dashboard

**Additional context**

### CODE REVIEW

**Comment by manchenkoff** [comment]
Hey @babanesma, thanks for the contribution! Can you please explain a bit more about how you got this warning?
The dashboard page is supposed to be a page that is available only for guests, but I couldn't reproduce this behavior by logging in.

**Comment by babanesma** [comment]
@manchenkoff I got this error when I pulled the repo code into my project 
On clean install it's not reproduced at all 

Although I am still confused on /dashboard because it holds `middleware: ['sanctum:auth', 'sanctum:verified'],` so it is supposed to be for logged in users only

**Comment by manchenkoff** [comment]
> Although I am still confused on /dashboard because it holds `middleware: ['sanctum:auth', 'sanctum:verified'],` so it is supposed to be for logged in users only

Yes, dashboard page (`/dashboard`) is only for authenticated users and the landing page (`/`) for guests only

File: nuxt.config.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ export default defineNuxtConfig({
   sanctum: {
     baseUrl: 'http://localhost:80',
     redirect: {
-      onGuestOnly: '/dashboard',
+      onLogin: '/dashboard',
```
Comment (manchenkoff) [inline nuxt.config.ts:33]: I believe that it is better to keep both then, `onLogin` is used to redirect after login, but `onGuestOnly` is used to redirect from pages that are supposed to be available for guests-only but user is already authenticated

Comment (babanesma) [reply nuxt.config.ts:34]: I added both options `onGuestOnly` and `onLogin`

**Pull request merged**